api/endpoint-tests/apds/activities/post.js

```diff
@@ -129,6 +129,7 @@ tap.test(
                     name: 'My Second Activity'
                   },
                   {
+                    id: Number,
```

Comment:
It turns out that tap's object pattern matcher understands the `Number` constructor.  :tada: